src/components/plugins/menu-item/menu-item.tsx

```diff
@@ -0,0 +1,30 @@
+import {h} from 'preact';
+import {ui} from '@playkit-js/kaltura-player-js';
+import * as styles from './menu-item.scss';
+import {A11yWrapper} from '@playkit-js/common/dist/hoc/a11y-wrapper';
+const {Icon, IconType} = ui.Components;
+
+interface MenuItemProps {
+  onClick: (player: any) => void;
+  pluginName: string;
+  icon: {svgUrl: string; viewBox: string};
+}
+
+const MenuItem = ({pluginName, icon, onClick}: MenuItemProps) => {
+  return (
+    // @ts-ignore - error TS2786: 'A11yWrapper' cannot be used as a JSX component.
```

Comment:
try upgrade version of `@playkit-js/common` the error should disappears

Reply:
done